packages/app-info/README.md

```diff
@@ -84,6 +85,18 @@ Get the type of cloud provider which is set to either Heroku or AWS depending on
 
 This is derived from some environment variables which we have high confidence are defined by only Heroku or AWS/Lambda
 
+### `appInfo.herokuAppId`
+
+Get the HEROKU_APP_ID which is the unique identifier for the application.e.g "9daa2797-e49b-4624-932f-ec3f9688e3da"
```

Comment:
**suggestion:** could we backtick the environment variable names so they're formatted as code blocks? Also maybe prefix with `process.env.` to match the documentation for the other properties. Same for the `appInfo.herokuDynoId` docs below.